kickstarts/almalinux-8-micro.aarch64.ks

```diff
@@ -75,22 +64,21 @@ echo '%_install_langs en_US.UTF-8' > /etc/rpm/macros.image-language-conf
 # force each container to have a unique machine-id
 > /etc/machine-id
 
-# create tmp directories because there is no tmpfs support in Docker
-umount /run
-systemd-tmpfiles --create --boot
-
-# disable login prompt and mounts
-systemctl mask console-getty.service \
-               dev-hugepages.mount \
-               getty.target \
-               systemd-logind.service \
-               sys-fs-fuse-connections.mount \
-               systemd-remount-fs.service
-
 # remove unnecessary files
-rm -f /var/lib/dnf/history.* \
+rm -rfv /var/lib/dnf \
       /run/nologin
-rm -fr /var/log/* \
+rm -rfv /var/log/* \
        /tmp/* /tmp/.* \
        /boot || true
 %end
+
+%post --nochroot --logfile=/mnt/sysimage/root/anaconda-post-nochroot.log --erroronfail
+set -eux
+
+# https://bugzilla.redhat.com/show_bug.cgi?id=1343138
+umount /mnt/sysimage/run
+# We don't have systemd and need to create manually
+chroot /mnt/sysimage install -d /run/lock -m 0755 -o root -g root
```

Comment:
Do we really need to chroot for that? root uid/gid will always be the same, so why not just create a `/mnt/sysimage/run/lock` directory similarly as you did on the next line?

Reply:
Agree. After these two line changes breaks `ks2rootfs` compatablity. until that available, use old or alternative

Reply:
It looks same without chroot.
```
bash-4.4# stat /run
  File: /run
  Size: 18              Blocks: 0          IO Block: 4096   directory
Device: 2fh/47d Inode: 17524580    Links: 3
Access: (0755/drwxr-xr-x)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2021-09-03 10:42:22.828981698 +0000
Modify: 2021-09-03 10:41:50.000000000 +0000
Change: 2021-09-03 10:42:22.822981670 +0000
 Birth: -
bash-4.4# stat /run/lock/
  File: /run/lock/
  Size: 6               Blocks: 0          IO Block: 4096   directory
Device: 2fh/47d Inode: 34884318    Links: 2
Access: (0755/drwxr-xr-x)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2021-09-03 10:42:22.828981698 +0000
Modify: 2021-09-03 10:41:50.000000000 +0000
Change: 2021-09-03 10:42:22.822981670 +0000
 Birth: -
```

Reply:
@srbala Oh! actual problem is `%post --nochroot` part right? :thinking: